doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -634,6 +634,88 @@ must be sent.
 *default location*, then the query will be declared as ambigous and an HTTP error response with code ``409``
 must be sent. 
 
+## Notification messages
+
+Notifications include two fields:
+
+* `subscriptionId` which represents the concerned subscription that originates the notification
+* `data` an array with the notification data itself which includes the entity and attributes concerned. Each
+  element in the array corresponds to a different entity. The way of representing each entity depends on
```

Comment:
By default each entity will be represented in normalized mode. However by using the 'attrsFormat' modifier it can be requested a simplified representation